lib/sgtn-client/loader/source.rb

```diff
@@ -22,11 +22,18 @@ def load_bundle(component, locale = nil)
         total_messages = {}
 
         (@source_bundle_path + component).glob('**/*.{yml, yaml}') do |f|
-          bundle = YAML.load(File.read(f))
-          total_messages.merge!(bundle&.first&.last) # TODO: Warning about inconsistent source locale
+          bundle = YAML.safe_load(File.read(f))
+          messages = bundle&.first&.last # TODO: Warn about inconsistent source locale
+          if messages.is_a?(Hash)
+            total_messages.merge!(messages)
+          else
```

Comment:
When will messages not be a Hash?

Reply:
Messages will be nil when the source file is empty.